packages/sync/src/class-defaults.php

```diff
@@ -894,7 +894,7 @@ public static function is_whitelisted_option( $option ) {
 		'promote_users',
 		'delete_themes',
 		'export',
-		'edit_comment',
+		'moderate_comments',
```

Comment:
see line 854 moderate_comments is already part of the whitelist.
so the change here doesn't add permissions it should be to simply drop edit_comment if not used.